pulseapi/users/utils.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from django.conf import settings
+from django.http import HttpResponseForbidden
+from django.views.generic.base import RedirectView
+
+from requests import post
+from requests.exceptions import RequestException
+
+# see https://developers.google.com/recaptcha/docs/verify
+RECAPTCHA_VERIFICATION_URL = 'https://www.google.com/recaptcha/api/siteverify'
+
+
+def verify_recaptcha(response_token):
+    """
+    Ask google to verify a client-generated recaptcha token
+    against what it supposedly generated.
+    """
+    try:
+        print(f"secret: {settings.RECAPTCHA_SECRET}")
```

Comment:
These print statements will be removed before landing, but you'll want them for debugging/testing.